bazel/repositories.bzl

```diff
@@ -562,6 +567,14 @@ def _io_opencensus_cpp():
         actual = "@io_opencensus_cpp//opencensus/exporters/trace/zipkin:zipkin_exporter",
     )
 
+def _grpc_java():
```

Comment:
This is a kind of terrifying dependency to take for just OpenCensus protos and C++. Is there any way you can partition this upstream in OpenCensus so that we don't need it?

Reply:
Why do we depend on java?

Reply:
From reading the OC cpp, it seems that it is needed for the proto https://github.com/census-instrumentation/opencensus-cpp/blob/8058a1b8efe6a63bd18673abc51223917d12d45d/WORKSPACE#L205-L210

Reply:
Yep, this is the opencensus-proto.

googleapis resolves this via the `switched_rules_by_language()` macro. Does opencensus-proto want to use the same approach?

Reply:
Removed dependency.